packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -128,8 +128,8 @@ contract OPContractsManager is ISemver, Initializable {
 
     // -------- Constants and Variables --------
 
-    /// @custom:semver 1.0.0-beta.14
-    string public constant version = "1.0.0-beta.14";
+    /// @custom:semver 1.0.0-beta.16
+    string public constant version = "1.0.0-beta.16";
```

Comment:
Using 16 because https://github.com/ethereum-optimism/optimism/pull/12169 bumps it to 15 and should be merged first